common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -482,6 +483,11 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     REPL_ADD_RAW_RESERVED_NAMESPACE("hive.repl.add.raw.reserved.namespace", false,
         "For TDE with same encryption keys on source and target, allow Distcp super user to access \n"
             + "the raw bytes from filesystem without decrypting on source and then encrypting on target."),
+    REPL_INCLUDE_EXTERNAL_TABLES("hive.repl.dump.include.external.tables", false,
```

Comment:
Name can be hive.repl.include.external.tables. Shall remove "dump"

Reply:
done